theme/brb/views/dataprivacy/privacy_logged_in.hbs

```diff
@@ -0,0 +1,13 @@
+{{#extend "lib/loggedin" hideSearch="true"}}
+
+    {{#content "styles" mode="append"}}
+        <link rel="stylesheet" href="/styles/imprint/imprint.css"/>
+    {{/content}}
+
+    {{#content "page"}}
+        {{#embed "dataprivacy/privacy" loggedin="true"}}{{/embed}}
+    {{/content}}
+
+    {{#content "scripts" mode="append"}}
```

Comment:
We should be consistent in the order of our content.
I think we normally use:
1. styles
2. scripts
3. page

Reply:
moved scripts above content